### DISCUSSION TITLE
请问怎么才能用mathjax渲染latex公式啊

### DISCUSSION THREAD

**Discussion by BEWINDOWEB** [ppoffice/hexo-theme-icarus#1166] [Q&A] [answered]
1、尝试过下载hexo-renderer-kramed插件，屏蔽默认render，但是怎么配都不行
https://github.com/sun11/hexo-renderer-kramed

2、从network看，mathjax的js文件都没加载，想尝试直接加在模板里面，发现是用jsx写的，我不懂前端，没法照着网上抄…
https://www.jianshu.com/p/0422327ce632

3、看了下这个issue，和我差不多的配置，但是我这个出不来
https://github.com/ppoffice/hexo-theme-icarus/issues/394

官网和icarus文档也没有介绍公式这块，求助作者大佬

![image](https://user-images.githubusercontent.com/6178098/210151101-764d0c4d-5fa3-4460-9b92-81c214a83ae2.png)
![image](https://user-images.githubusercontent.com/6178098/210151103-34aadeea-2975-46ef-ac17-971b5b6734ab.png)
![image](https://user-images.githubusercontent.com/6178098/210151109-3f1e2fc6-28ac-4022-9281-2b78f3c2bc59.png)
![image](https://user-images.githubusercontent.com/6178098/210151163-876c6363-0866-4d2f-affd-70449fe3bc49.png)

**Comment by ppoffice** [comment, marked as answer]
@BEWINDOWEB https://ppoffice.github.io/hexo-theme-icarus/Plugins/Other/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E5%85%B6%E4%BB%96%E6%8F%92%E4%BB%B6/#MathJax

**Reply by BEWINDOWEB** [reply]
感谢
